response_operations_ui/templates/social-view-case-details.html

```diff
@@ -24,5 +24,22 @@ <h2 class="neptune">Property address</h2>
             </dl>
         </section>
     </div>
+    <section class="social-case-section u-mt-l">
+        <h2 class="neptune">Unique access code</h2>
+        <dt class="new-iac__count" id="new-iac-count">
+            There have been {{iac_count}} unique access codes issued for this case.
+        </dt>
+            <form action="{{ url_for('social_bp.generate_iac') }}" method='POST'>
+                <input type="hidden" name="case_id" value="{{case_id}}"/>
+                <button type="submit" class="btn btn--primary generate-iac-submit" id="generate-iac-submit">Generate UAC</button>
+            </form>
```

Comment:
Can we change this to `btn btn--secondary btn--small` to match the prototype and pattern library